acpdr/swagger-specs/schema-registry.yaml

```diff
@@ -775,15 +924,13 @@ paths:
         - $ref: '#/parameters/createDataTypePayload'
       responses:
         '201':
-          description: Returns details of the newly created data type, including $id, meta:altId, and version.
+          description: A successful response returns the details of the newly created data type, including $id, meta:altId, and version.
```

Comment:
Same comment about removing that end blurb, and just cutting it off at "... newly created data type."